jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -130,16 +130,16 @@ public void apply(Project project) {
             Task dependsOnTask;
             if (warTask != null) {
               ExplodedWarTask explodedWarTask =
-                  (ExplodedWarTask)
-                      project
-                          .getTasks()
-                          .create(EXPLODED_WAR_TASK_NAME, ExplodedWarTask.class)
-                          .dependsOn(warTask);
+                  project.getTasks().create(EXPLODED_WAR_TASK_NAME, ExplodedWarTask.class);
+              explodedWarTask.dependsOn(warTask);
               explodedWarTask.setWarFile(warTask.getArchivePath().toPath());
               explodedWarTask.setExplodedWarDirectory(
                   GradleProjectProperties.getExplodedWarDirectory(projectAfterEvaluation));
               // Have all tasks depend on the 'jibExplodedWar' task.
               dependsOnTask = explodedWarTask;
+            } else if ("packaged".equals(jibExtension.getContainerizingMode())) {
```

Comment:
We process this in plugins-common to not be case-sensitive, right? So it could be possible that someone configures "PACKAGED", and this wouldn't depend on the jar task, and could cause the build to fail?

Reply:
You have a point. I made `getContainerizingModeChecked()` case-sensitive.